modules/mod_ginger_base/mod_ginger_base.erl

```diff
@@ -168,9 +168,7 @@ manage_schema(_Version, Context) ->
                 ]}
             ]}
         ]
-    },
-    z_datamodel:manage(?MODULE, Datamodel, Context),
-    schema:create_identity_if_not_exists(editor_dev, "redacteur", "redacteur", Context).
```

Comment:
Proposal to remove the automatic creation of `editor_dev`. If we need one we should provide it in fixtures.

Reply:
Ok, I've never used it, maybe @loetie has

Reply:
There is also an `manage_data/2` function. That is called _after_ the `manage_schema/2` has been handled.

See https://test.zotonic.com/page/1353/modules

```erlang
manage_data(_Version, Context) ->
    %% Whatever data needs to be installed after the datamodel
    %% has been installed.
    ok.
```

Reply:
Good point, thanks.

Reply:
BTW `schema` is a dangerous name for a module. I can imagine that other projects will also try to use this generic name.  Better prefix it with `ginger_`, as it is a Ginger-only module.

Reply:
Yes, I completely agree, that struck me too.